product_sold_by_delivery_week/models/sale_order.py

```diff
@@ -16,8 +16,9 @@ def _compute_weekly_sold_delivered_shown(self):
         _format_weekly_string = self.env["product.product"]._format_weekly_string
         self.weekly_sold_delivered_shown = False
         products = self.mapped("product_id").filtered(lambda x: x.type != "service")
+        partners = self.mapped("order_id.partner_id.commercial_partner_id.child_ids")
```

Comment:
This don't work in single partner, child_ids field don't contain to himself

Reply:
It could be fixed with something like:

```suggestion
        partners = order_id.partner_id | self.mapped("order_id.partner_id.commercial_partner_id.child_ids")
```